src/AdamQuaile/Behat/CommandRunnerExtension/CommandRunnerSubscriber.php

```diff
@@ -81,7 +81,9 @@ private function runInBackgroundForHook(Process $process, $hook)
      */
     private function buildProcess($command)
     {
-        return new Process($command);
```

Comment:
Thanks for the pull request. I think there might be some escaping issues with splitting on any space, e.g. 

```cp 'file 1' 'file 2'```

If a user passes in a command as a string that's already escaped then the array would be `["cp", "'file", "1'", "'file", "2'"]` which symfony will then escape. 

Could you try with `Process::fromShellCommandline` as described in https://symfony.com/doc/current/components/process.html#using-features-from-the-os-shell and see if that works?

Reply:
Right ! I have changed an used `Process::fromShellCommandline` which works fine